src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandlerTest.java

```diff
@@ -251,6 +251,30 @@ public void givenDirectionTypeIsNull_displayAnError() {
         assertEquals("Direction Type cannot be empty", response.getErrors().toArray()[0]);
     }
 
+    @SuppressWarnings({"Indentation", "unused"})
+    private Object[] getDirectionNoticeConfidentialMembers() {
+        return new Object[]{
+                new Object[]{DirectionType.APPEAL_TO_PROCEED, "general", null, DIRECTION_ACTION_REQUIRED},
+                new Object[]{DirectionType.APPEAL_TO_PROCEED, "confidential", List.of("fta"), DwpState.DIRECTION_ACTION_REQUIRED},
+                new Object[]{DirectionType.APPEAL_TO_PROCEED, "confidential", List.of("appellantOrAppointee"), null},
+                new Object[]{DirectionType.PROVIDE_INFORMATION, "general", null, DIRECTION_ACTION_REQUIRED},
+                new Object[]{DirectionType.PROVIDE_INFORMATION, "confidential", List.of("fta"), DwpState.DIRECTION_ACTION_REQUIRED},
+                new Object[]{DirectionType.PROVIDE_INFORMATION, "confidential", List.of("appellantOrAppointee"), null},
+        };
+    }
+
+    @Test
+    @Parameters(method = "getDirectionNoticeConfidentialMembers")
+    public void givenDirectionNoticeCheckFtaStateBasedOnConfidentiality(DirectionType directionType, String confidentialityType, List<String> chosenMembers, DwpState newFtaState) {
+        callback.getCaseDetails().getCaseData().setDirectionTypeDl(new DynamicList(directionType.toString()));
+        callback.getCaseDetails().getCaseData().setConfidentialityType(confidentialityType);
+        callback.getCaseDetails().getCaseData().setConfidentialityPartyMembers(chosenMembers);
+        sscsCaseData.setDirectionDueDate("11/07/2025");
```

Comment:
Does this date have significance? Could we LocalDate.now() it? there's a few examples in other tests where we're inputting dates if you want to use those as jumping off point.